.github/workflows/cancel.yml

```
@@ -13,6 +13,6 @@ jobs:
       actions: write  # for styfle/cancel-workflow-action to cancel/stop running workflows
     runs-on: ubuntu-latest
     steps:
-    - uses: styfle/cancel-workflow-action@0.10.0
+    - uses: styfle/cancel-workflow-action@0.10.1
```

```suggestion
    - uses: styfle/cancel-workflow-action@0.10
```
Wouldn't this be better to not have to update for every bugfix release?

---

I've switched other dependabot schedules to monthly to avoid some of this.